prow/gerrit/adapter/adapter.go

```diff
@@ -332,9 +376,18 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 
 	switch change.Status {
 	case client.Merged:
-		// TODO: Do we want to add support for dynamic postsubmits?
-		postsubmits := c.config().PostsubmitsStatic[cloneURI.String()]
-		postsubmits = append(postsubmits, c.config().PostsubmitsStatic[cloneURI.Host+"/"+cloneURI.Path]...)
+		postsubmits := []config.Postsubmit{}
+		if cache != nil {
+			postsubmits, err = cache.GetPostsubmits(cloneURI.Host+"/"+cloneURI.Path, func() (string, error) { return baseSHA, nil })
```

Comment:
Nit: you should do something like

```
hostPath := cloneURI.Host+"/"+cloneURI.Path
```

and use this variable here as an argument to `GetPostsubmits`. As it stands it's hard to talk about this argument as it is an unevaluated expression.

Reply:
Done